src/Compilers/Core/Portable/CodeGen/SwitchIntegralJumpTableEmitter.cs

```diff
@@ -466,7 +466,20 @@ private void EmitRangeCheckedBranch(ConstantValue startConstant, ConstantValue e
             }
             else
             {
-                _builder.EmitIntConstant(endConstant.Int32Value - startConstant.Int32Value);
+                int Int32Value(ConstantValue value)
```

Comment:
Do we need to shim `Int64Value` in the code above the same way?

Basically, if the testcase is changed slightly to be `       void RunTest(ulong testUlong)` with everything else the same, do we still have a problem?

I think we do not have a problem since labels will be statically promoted to `ulong`.
Probably worth adding a testcase though.